docs/verein360-enpoint.md

```diff
@@ -0,0 +1,158 @@
+Verein360/BLSV should be able to send already approved entitlementcard application for their trainers.
+An api token that can be generated from the "Digitale Druckerei" account is required to access the endpoint.
+All details about the endpoint can be found in the [API Documentation](../specs/backend-api.graphql).
+
+## Usage
+
+### Set Authourization Header
+
+Set the HTTP header `Authorization Bearer <api-token>` to access the endpoint.
+
+### Call GraphQL Mutation to add an application
+
+```graphql
+mutation addEakApplication($regionId: Int!, $application: ApplicationInput!, $project: String!) {
+  result: addEakApplication(regionId: $regionId, application: $application, project: $project)
+}
+```
+
+### Sample Mutation Variables
+
+```json
+  {
+  "regionId": 2,                                      // Must be the correct region ID where the card owner lives
+  "application": {
+    "applicationDetails": {
+      "applicationType": "FIRST_APPLICATION",         // Must be FIRST_APPLICATION
+      "cardType": "BLUE",                             // Must be BLUE
+      "givenInformationIsCorrectAndComplete": true,   // Must be true
+      "hasAcceptedEmailUsage": true,                  // true or false
+      "hasAcceptedPrivacyPolicy": true,               // Must be true
+      "wantsDigitalCard": true,                       // Must be true
+      "wantsPhysicalCard": false,                     // Must be false
+      "blueCardEntitlement": {
+        "entitlementType": "WORK_AT_ORGANIZATIONS",   // Must be WORK_AT_ORGANIZATIONS
+        "workAtOrganizationsEntitlement": {
+          "list": {
+            "amountOfWork": 7.5,
+            "organization": {
+              "address": {
+                "country": {
+                  "shortText": "Germany"
+                },
+                "houseNumber": {
+                  "shortText": "123"
+                },
+                "location": {
+                  "shortText": "Munich"
+                },
+                "postalCode": {
+                  "shortText": "80331"
+                },
+                "street": {
+                  "shortText": "Example Street"
+                }
+              },
+              "category": {"shortText": "Sport"},     // Must be Sport
+              "contact": {
+                "email": {"email": "jane.doe@sportverein.de"},
+                "hasGivenPermission": true,           // Must be true
+                "name": {"shortText": "Jane Doe"},
+                "telephone": {"shortText": "0150123456789"}
+              },
+              "name": {
+                "shortText": "Sportverein Augsburg-Nord"
+              }
+            },
+            "payment": false,                         // Must be false
+            "responsibility": {"shortText": "Trainer"},
+            "workSinceDate": {"date": "2020-10-06"},
+            "isAlreadyVerified": true                 // Must be true, so that we can mark the application as verified. 
```

Comment:
Please let me know what we want to do in this case.
We could
1) Send no error message and just accept the applicaiton and not mark it as verified
2) Send an unauthorized error message and not save the application

I think I would prefer 2) to ensure verein360 fixes auth problems and not make the municipalities a lot of work of verifying applications manullay.

Reply:
But as we discussed this source may also be defined via the purpose of the bearer token or a particular role for verein 360. We may add some meta data for API Tokens in general